.github/workflows/dart.yml

```
@@ -1,13 +1,30 @@
+
 on:
   pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.coderabbit.yaml'
+      - 'LICENSE'
+      - '.gitignore'
   push:
-    branches: [ main ]
+    branches: [ main, "release/**" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)